operate/client/e2e-playwright/tests/batchMoveModification.spec.ts

```diff
@@ -50,186 +49,179 @@ test.beforeAll(async ({request}) => {
   );
 });
 
-(IS_BATCH_MOVE_MODIFICATION_ENABLED ? test.describe : test.describe.skip)(
```

Comment:
I only changed this to `test.describe`. The rest should only be indentation changes.